src/layers/core/scatterplot-layer/scatterplot-layer.js

```diff
@@ -65,6 +67,16 @@ export default class ScatterplotLayer extends Layer {
   }
 
   willReceiveProps(oldProps, newProps) {
+    if (oldProps.outlineMode !== newProps.outlineMode) {
+      const {gl} = this.state;
+      const model = this._getModel(gl, newProps.outlineMode ? 'LINE_LOOP' : 'TRIANGLE_FAN');
```

Comment:
I'd prefer to use GL constants here. `GL.LINE_LOOP` and `GL.TRIANGLE_FAN`. `GL` can be imported from luma, see other layers on dev (if examples are needed). 

The strings will be looked up and resolved to those constants, but no need to make the reader guess what is happening.